pkgs/development/python-modules/pyghmi/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  wheel,
+  cryptography,
+  python-dateutil,
+  six,
+  pbr,
+  coverage,
+  oslotest,
+  stestr,
+}:
+
+buildPythonPackage rec {
+  pname = "pyghmi";
+  version = "1.5.76";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
Please fetch from github

Reply:
I wasn't able to find any actively maintained GitHub mirror for this package, so maybe we still need to rely on PyPI for this package.